README.md

```diff
@@ -102,8 +102,8 @@ jobs:
     strategy:
       matrix:
         operating-system: [ ubuntu-latest, windows-latest ]
-        php: [ '7.4', '8.0' ]
-        symfony: ['^4.4', '^5.3']
+        php: [ '7.4', '8.0', '8.1' ]
+        symfony: ['^4.4', '^5.3', '^5.4', '^6.0', '^6.1']
```

Comment:
either `5.3.*` or `~5.3.0`.
But you using caret in minor version does not make sens: '^5.3' intersects with '^5.4'

Reply:
@jderusse Thanks.